### DISCUSSION TITLE
help with user installs

### DISCUSSION THREAD

**Discussion by Bobler6** [Pycord-Development/pycord#2569] [Q&A]
Hey, I can't figure out how to do user installs for pycord. Help is appreciated!

**Comment by Icebluewolf** [comment]
Please reference this example: https://github.com/Pycord-Development/pycord/blob/master/examples/app_commands/slash_users.py

You can also reference the [discord API tutorial](https://discord.com/developers/docs/tutorials/developing-a-user-installable-app#choosing-supported-installation-contexts) for how to set up the bot in the developer portal. I linked where you should start reading and you should stop reading at "Step 2: Setting Up Commands" as the other parts of the page are not relevant if you are using py-cord.

You can also join the Discord for quicker help: https://discord.gg/pycord
